packages/definitions/src/integration.ts

```diff
@@ -49,6 +49,12 @@ export const integrationDefs = {
     iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/png/qbittorrent.png",
     category: ["downloadClient", "torrent"],
   },
+  aria2: {
+    name: "Aria2",
+    secretKinds: [["apiKey"]],
+    iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/png/aria2.png",
```

Comment:
This gives me ``Package size exceeded the configured limit of 50 MB. Try https://github.com/homarr-labs/dashboard-icons/tree/master/png/aria2.png instead.``

Reply:
Yeah the icon does not exist and has to be added to the dashboard-icons repository

Reply:
Aria2 don't have any offical icons. We can use the one below as it icon, or if you find any better, please help add to the repo.
![aria2](https://github.com/user-attachments/assets/16ef48da-d0da-43c0-9b1e-60e066ccd477)

Reply:
![aria2](https://github.com/user-attachments/assets/bdadbbc2-f9aa-4052-977a-6f1009829556)

I would suggest we use the above one. It is an svg